### DISCUSSION TITLE
[Feature] Option to enable/disable Machine Learning for each user in Administration

### DISCUSSION THREAD

**Discussion by upsetdog** [immich-app/immich#9961] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### Problem description

I have a two Immich accounts on my server, one is actively used (where face/object detection is useful), and another is for a large archive of old photos which doesn't need any ML features (it'd be a waste of resources and take way too long to run it through ML) - if that makes sense.

### Feature description

Just a toggle switch to enable/disable (enabled by default) machine learning per user in the "Edit User" modal under /admin/user-management.

Thank you :)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by mmomjian** [comment]
#9089
